manifests/android.pp

```diff
@@ -0,0 +1,10 @@
+class hieradata::android (
+	Array[String] $sdk_packages, 
+)
+{
+  $sdk_packages.each |String $sdk_package| {
+    exec { "install_android_sdk_package_${sdk_package}":
+      command => "cmd.exe /c sdkmanager --install \"${sdk_package}\"",
```

Comment:
This is not idempotent, and hieradata is not the best place for a class, which looks like a classic profile, but it's just a naming thing

Reply:
Agree. 
@alvagante What do you think if we implement something like:
if (is android package installed?) == false {
     exec install command
}

Reply:
@olegy-ait a cleaner solution would be to use the unless/onlyif  or the creates parameters of the exec resource in order to detect if the package is already installed before running the exec that installs it